### PR TITLE
Check for successful messages in the responses when uploading to Chrome Web Store

### LINKED ISSUE #49: The publish build step reports a success even though publish failed
https://github.com/GoogleChromeLabs/form-troubleshooter/runs/3282260568?check_suite_focus=true

```json
{
  "kind": "chromewebstore#item",
  "id": "***",
  "uploadState": "FAILURE",
  "itemError": [
    {
      "error_code": "ITEM_NOT_UPDATABLE",
      "error_detail": "The item cannot be updated now because it is in pending review, ready to publish, or deleted status."
    }
  ]
}
```

```json
{
  "error": {
    "code": 400,
    "message": "Publish condition not met: ",
    "errors": [
      {
        "message": "Publish condition not met: ",
        "domain": "chromewebstore.access",
        "reason": "badRequest"
      }
    ]
  }
}
```

### PR DESCRIPTION
**Pull request by socsieng**
The Chrome Web Store APIs always return a 200 status code even if there is an error. This change inspects the response message to see if the request was actually successful.

Fixes #49

### CODE REVIEW

File: scripts/publish.sh [hunk 1]
```diff
@@ -45,5 +45,24 @@ npm run build
 (cd build && zip -r extension.zip ./)
 
 # upload and publish extension
-curl -H "Authorization: Bearer ${access_token}" -H "x-goog-api-version: 2" -X PUT -T build/extension.zip "https://www.googleapis.com/upload/chromewebstore/v1.1/items/${EXTENSION_ID}"
-curl -H "Authorization: Bearer ${access_token}" -H "x-goog-api-version: 2" -H "Content-Length: 0" -X POST "https://www.googleapis.com/chromewebstore/v1.1/items/${EXTENSION_ID}/publish"
+response=`curl -H "Authorization: Bearer ${access_token}" -H "x-goog-api-version: 2" -X PUT -T build/extension.zip "https://www.googleapis.com/upload/chromewebstore/v1.1/items/${EXTENSION_ID}" 2>/dev/null`
+echo "$response"
+status=`echo "console.log($response.uploadState)" | node`
+
+if [[ $status != "SUCCESS" ]]
```
Comment (samdutton) [inline scripts/publish.sh:52]: Nit:  I think this should be `!==` (following [Google style](https://google.github.io/styleguide/jsguide.html#features-equality-checks)).

File: scripts/publish.sh [hunk 2]
```diff
@@ -45,5 +45,24 @@ npm run build
...
+  echo "Failed to upload extension. Response:"
+  echo "$response"
+  exit 1
+fi
+
+response=`curl -H "Authorization: Bearer ${access_token}" -H "x-goog-api-version: 2" -H "Content-Length: 0" -X POST "https://www.googleapis.com/chromewebstore/v1.1/items/${EXTENSION_ID}/publish" 2>/dev/null`
+echo "$response"
+status=`echo "console.log($response.status[0])" | node`
+
+if [[ $status != "OK" ]]
```
Comment (samdutton) [inline scripts/publish.sh:63]: Same here: I think this should be `!==`

**Comment by samdutton** [comment]
LGTM % nits.

File: scripts/publish.sh (hunk 1 above)
Comment (socsieng) [reply scripts/publish.sh:52]: This is a shell script, not JavaScript so `!==` isn't valid here.

File: scripts/publish.sh (hunk 2 above)
Comment (socsieng) [reply scripts/publish.sh:63]: This is a shell script, not JavaScript so `!==` isn't valid here.

Comment (samdutton) [reply scripts/publish.sh:63]: And again...

**Pull request merged**